src/common/devtools.js

```diff
@@ -145,6 +150,25 @@ const report =
     )
   ];
 
+const snapshot =
+  (model) => {
+    const id = Date.now().toString(32)
+    const message =
+      [ `\n\n`
+        , `echo '${JSON.stringify(window.application.model.value)}' > `
+        , `./dist/snapshot-${id}.json`
```

Comment:
> servo -w -b --pref dom.mozbrowser.enabled "file:///Users/paul/git/browserhtml/dist/index.html?replay=/tmp/snapshot-1af86t6v7.json"

Hmm, I would expect XHR to be disallowed in that case.

Anyway I'll just dump JSON ;)